JS/ReviewSession12/app__generate-Password/js/script.js

```diff
@@ -0,0 +1,55 @@
+"use strict";
+
+const range = document.querySelector(".form-range");
+let showLengthPassword = document.querySelector(".length");
+const generatePassword = document.querySelector(".generate_password");
+const numbers = document.getElementById("numbers");
+const lowerCaseLetter = document.getElementById("lower_case");
+const upperCaseLetter = document.getElementById("uppercase_letters");
+const passwordShow = document.getElementById("Password_show");
+
+let str = [];
+let string = "";
+
+//    function generate
+function* generateSequence(start, end) {
+  for (let i = start; i <= end; i++) yield i;
+}
+
+function* generatePasswordCodes() {
+  // 0..9
+  if (numbers.checked) {
+    yield* generateSequence(48, 57);
+  }
+  // A..Z
+  if (upperCaseLetter.checked) {
+    yield* generateSequence(65, 90);
+  }
+  // a..z
+  if (lowerCaseLetter.checked) {
+    yield* generateSequence(97, 122);
+  }
+}
+
+function randoomInt() {
+  return Math.random() - 0.5;
```

Comment:
[Is Math.random() cryptographically secure?](https://stackoverflow.com/questions/5651789/is-math-random-cryptographically-secure)